cmd/controller/app/options/options.go

```diff
@@ -285,6 +295,10 @@ func (o *ControllerOptions) Validate() error {
 		return fmt.Errorf("invalid default issuer kind: %v", o.DefaultIssuerKind)
 	}
 
+	if float32(o.KubernetesAPIBurst) < o.KubernetesAPIQPS {
+		return fmt.Errorf("invalid value for kubernetes-api-burst: %v must be higheror equal to kubernetes-api-qps: %v", o.KubernetesAPIQPS, o.KubernetesAPIQPS)
```

Comment:
`higher or`. Also, can these be negative numbers? I guess something else will error elsewhere if they are :)

Reply:
i think it will... happy to validate :D